contracts/BSCValidatorSet.sol

```diff
@@ -341,63 +372,54 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
 
   /*********************** For slash **************************/
   function misdemeanor(address validator)external onlySlash override{
-    uint256 index = currentValidatorSetMap[validator];
-    if (index <= 0) {
-      return;
+    uint256 validatorIndex = _misdemeanor(validator);
```

Comment:
We need change the logic of slash contract, to ensure no one can jump `misdemeanor`

Reply:
fixed